features/order/managing_orders/seeing_orders_currency_on_index.feature

```diff
@@ -26,31 +26,3 @@ Feature: Seeing the currency in which all orders have been placed
         And the customer chose "DHL" shipping method with "Cash on Delivery" payment
         When I browse orders
         Then I should see the order "#00000666" with total "$40.00"
-
-    @ui
-    Scenario: Seeing an order placed in a currency other than the base
-        Given there is a customer "satin@teamlucifer.com" that placed an order "#00000666"
-        And the customer has chosen to order in the "GBP" currency
-        And the customer bought a single "Angel T-Shirt"
-        And the customer "No Face" addressed it to "Lucifer Morningstar", "Seaside Fwy" "90802" in the "United States"
-        And for the billing address of "Mazikeen Lilim" in the "Pacific Coast Hwy", "90806" "Los Angeles", "United States"
-        And the customer chose "DHL" shipping method with "Cash on Delivery" payment
-        When I browse orders
-        Then I should see the order "#00000666" with total "£40.00"
-
-    @ui
-    Scenario: Seeing multiple orders in different currencies
```

Comment:
We should have such scenario, but use 2 channels with different base currencies.